packages/worker/src/runtime.ts

```diff
@@ -435,4 +434,35 @@ export class Runtime {
       return undefined;
     }
   }
+
+  /**
+   * Create an ephemeral Temporal server.
+   *
+   * Hidden since it is meant to be used internally by the testing framework.
+   * @hidden
+   */
+  public async createEphemeralServer(options: native.EphemeralServerConfig): Promise<native.EphemeralServer> {
```

Comment:
[Minor/Style] This should be moved closer to other create/close resources methods higher in the file. They share some important semantic (`pendingCalls`, `backRefs`) which would be easier to maintain if all these are gathered.

Reply:
I'm refactoring this whole class